### PR TITLE
UI: make header

### PR DESCRIPTION
**Pull request by harshbrahmxatriya**
Adding the sliding menu bar.
How to test:
Go to URL:https://deploy-preview-45--wehelpvip.netlify.app/
Part of UI - header

### CODE REVIEW

File: components/layout/header.vue [hunk 1]
```diff
@@ -69,23 +68,46 @@
...
+                  <option>RUS</option>
+                  <option>UKR</option>
+                </select>
+              </div>
+              <br />
+              <div class="h-20"></div>
+              <br />
+              <div class="float-right m-8 space-y-2">
+                <a href="https://t.me/wehelplive">
+                  <Icon icon="akar-icons:telegram-fill" height="24" width="24"
```
Comment (VOlekh) [inline components/layout/header.vue:96]: In the footer, I use another icon for social networks. How we can make them consistent?

File: components/layout/header.vue [hunk 2]
```diff
@@ -116,11 +138,11 @@ import {
   XIcon,
 } from '@heroicons/vue/outline'
 import { ChevronDownIcon } from '@heroicons/vue/solid'
-
+import { Icon } from '@iconify/vue'
 const navigation = [
-  { name: 'Find Accomodation', to: '/guest' },
-  { name: 'UAH Exchange', to: '/exchange' },
-  { name: 'Recommend', to: '/add' },
+  { name: 'Add service', to: '/guest' },
```
Comment (VOlekh) [inline components/layout/header.vue:143]: Not sure that links(to:/) correspond to the names of menu items

File: components/layout/header.vue [hunk 3]
```diff
@@ -69,23 +68,46 @@
...
           </div>
           <div class="py-6 px-5 space-y-6">
-            <div class="grid grid-cols-2 gap-y-4 gap-x-8">
+            <div class="flex flex-col float-right space-y-4">
               <NuxtLink
                 v-for="item in navigation"
                 :key="item.name"
                 :to="item.to"
-                class="text-base font-medium text-gray-900 hover:text-gray-700"
+                class="text-bas font-light text-gray-900 hover:text-gray-700"
```
Comment (VOlekh) [inline components/layout/header.vue:76]: Will need to change text color to #202020 according to prototype

File: components/layout/header.vue [hunk 4]
```diff
@@ -9,7 +9,8 @@
             <BrandLogo class="w-24 h-24" />
           </NuxtLink>
         </div>
-        <div class="-mr-2 -my-2 md:hidden">
+        <div class="font-medium text-gray-800">Hotline 24/7</div>
```
Comment (VOlekh) [inline components/layout/header.vue:12]: According to the prototype layout of the slide over panel of the menu is longer  - takes the whole side of the screen in mobile mode

Comment (VOlekh) [inline components/layout/header.vue:12]: And need a shadow for the top bar.

File: components/layout/header.vue [hunk 5]
```diff
@@ -9,7 +9,8 @@
...
+        <div class="mr-2 my-2 md:hidden">
```
Comment (VOlekh) [inline components/layout/header.vue:13]: But I really like how it works and look:)

**Review by VOlekh — APPROVED** [review]
Some changes in links, colors, text, and length of popover sidebar.

File: components/layout/header.vue (hunk 2 above)
Comment (harshbrahmxatriya) [reply components/layout/header.vue:143]: Yeah I told you that I need the links for those texts like 'Add service'

File: components/layout/header.vue (hunk 4 above)
Comment (harshbrahmxatriya) [reply components/layout/header.vue:12]: Okay I should see that in mobile phone for testing

File: components/layout/header.vue (hunk 5 above)
Comment (harshbrahmxatriya) [reply components/layout/header.vue:13]: Thank you

File: components/layout/header.vue (hunk 4 above)
Comment (harshbrahmxatriya) [reply components/layout/header.vue:12]: Okay

File: components/layout/header.vue (hunk 3 above)
Comment (harshbrahmxatriya) [reply components/layout/header.vue]: Okay,  now it's containing that color

File: components/layout/header.vue (hunk 1 above)
Comment (harshbrahmxatriya) [reply components/layout/header.vue:98]: I looked into the footer code, I think this one is a lot simpler to use. What say?

File: components/layout/header.vue [hunk 6]
```diff
@@ -116,11 +142,11 @@ import {
...
 } from '@heroicons/vue/outline'
 import { ChevronDownIcon } from '@heroicons/vue/solid'
-
+import { Icon } from '@iconify/vue'
 const navigation = [
-  { name: 'Find Accomodation', to: '/guest' },
-  { name: 'UAH Exchange', to: '/exchange' },
-  { name: 'Recommend', to: '/add' },
+  { name: 'Add service', to: '/add' },
+  { name: 'Partnership', to: '/' },
```
Comment (razbakov) [inline components/layout/header.vue:148]: please create a new page for Partnership and Contact

**Review by razbakov — CHANGES_REQUESTED** [review]

**Comment by razbakov** [comment]
@harshbrahmxatriya header have to be full width, please use TailwindUI code for reference.

File: components/layout/header.vue (hunk 6 above)
Comment (harshbrahmxatriya) [reply components/layout/header.vue:148]: Okay

**Comment by harshbrahmxatriya** [comment]
> @harshbrahmxatriya header have to be full width, please use TailwindUI code for reference.

Yeah actually I reverted to the previous code, now it is full width which I tested on mobile phone also

File: components/layout/header.vue [hunk 7]
```diff
@@ -68,24 +67,51 @@
...
-                Need help?
-              </a>
+              <div class="h-5"></div>
+              <div class="m-6">
+                <select
+                  class="form-select appearance-none rounded-lg block w-full px-3 py-1.5 text-base font-normal text-gray-700 bg-white bg-clip-padding bg-no-repeat border border-solid border-gray-300 rounded transition ease-in-out m-0 focus:text-gray-700 focus:bg-white focus:border-blue-600 focus:outline-none"
+                  id="language"
+                >
+                  <option selected>EN</option>
+                  <option>GER</option>
```
Comment (razbakov) [inline components/layout/header.vue:88]: Language names are inconsistent. Please use 2-characters name, i.e. DE, RU, UA, etc.

File: components/layout/header.vue (hunk 6 above)
Comment (razbakov) [reply components/layout/header.vue:148]: Why did you resolve this conversation? The link is still not updated here.

**Review by razbakov — CHANGES_REQUESTED** [review]

File: components/layout/header.vue (hunk 7 above)
Comment (harshbrahmxatriya) [reply components/layout/header.vue]: Done!

File: components/layout/header.vue (hunk 6 above)
Comment (harshbrahmxatriya) [reply components/layout/header.vue]: I've added it

**Comment by razbakov** [comment]
@harshbrahmxatriya I marked your PR as draft. Please add me to reviewers and make it "Ready for review" when it is ready.

**Comment by razbakov** [comment]
@harshbrahmxatriya 
- This branch has conflicts that must be resolved
- Fonts are not the same as on design

**Review by razbakov — CHANGES_REQUESTED** [review]
@harshbrahmxatriya 
- This branch has conflicts that must be resolved
- Fonts are not the same as on design

**Comment by razbakov** [comment]
@harshbrahmxatriya why did you close it?

**Comment by harshbrahmxatriya** [comment]
> @harshbrahmxatriya why did you close it?

There were many mix-ups while working on the same, so for simplicity I deleted the branch and created from main again and pasted the code of header component into it

**Comment by razbakov** [comment]
@harshbrahmxatriya have you heard about force push?

https://superuser.com/questions/716818/how-to-overwrite-a-branch-in-git-with-master

Also search in Google:
- How to force push branch in git?
- How to rewrite commit history in my branch?

Please don't delete branches, don't close pull requests. Work on those that you already started. There is no way back. Only forward.

**Pull request closed without merging**